templates/template/README.md

```diff
@@ -79,24 +78,103 @@ Before submitting any PRs, a Github Workflow will check to see if your package d
+
+You'll also want to update the `packageVersion` and `releaseCandidateVersion` located in `packages/${PACKAGE}/package.yaml`. See the section below for how to update this field.
+
+Once you have made those changes, the Workflow will be:
+```shell
+make prepare # Instantiates the chart in the workingDir specified in the package.yaml
+# Make your changes here to the workingDir directly here
+make patch # Saves changes to generated-changes/
+make clean # Cleans up your workingDir, leaving behind only the generated-changes/
+```
+
+Once your directory is clean, you are ready to submit a PR.
+
+#### Versioning Packages
+
+If this major.minor.patch version of the Chart has never been released, reset the `packageVersion` to `01` and the `releaseCandidateVersion` to `00`.
+
+If this major.minor.patch version of the Chart has been released before:
+- If this is the first time you are making a change to this chart for a specific Rancher release (e.g. the current `packageVersion` has already been released in the Live Branch), increment the `packageVersion` by 1 and reset the `releaseCandidateVersion` to `00`.
+- Otherwise, increment the `releaseCandidateVersion` by 1. Leave the `packageVersion` as is.
+
+{{ end -}}
+
+{{- if (eq .Template "staging") }}
+### Cutting a Release
+
+In the Staging branch, cutting a release involves moving the contents of the `assets/` directory into `released/assets/`, deleting the contents of the `charts/` directory, and updating the `index.yaml` to point to the new location for those assets. This process is entirely automated via the `make release` command.
+
+Once you successfully run the `make release` command, ensure the following is true:
+- The `assets/` and `charts/` directories each only have a single file contained within them: `README.md`
+- The `released/assets/` directory has a .tgz file for each releaseCandidateVersion of a Chart that was created during this release.
+- The `index.yaml` and `released/assets/index.yaml` both are identical and the `index.yaml`'s diff shows only two types of changes: a timestamp update or a modification of an existing URL from `assets/*` to `released/assets/*`.
+
+No other changes are expected.
+
+Note: these steps should be taken only after following the steps to cut a release to your Live Branch.
+{{ end -}}
+
+{{- if (eq .Template "live") }}
+### Cutting a Release
+
+In the Live branch, cutting a release requires you to run the `make sync` command. This command will automatically get the latest charts / resources merged into the the branches you sync with (as indicated in this branch's `configuration.yaml`) and will fail if any of those branches try to modify already released assets. If the `make sync` command fails, you might have to manually make changes to the contents of the Staging Branch to resolve any issues.
```

Comment:
nit: i'll stop making this comment to avoid spam, but I think creating a newline per new sentence is great for readability and editing.

Reply:
addressed in latest, thanks for catching!